response/core/serializers.py

```diff
@@ -124,5 +125,8 @@ def update(self, instance, validated_data):
         instance.summary = validated_data.get("summary", instance.summary)
         instance.severity = validated_data.get("severity", instance.severity)
 
+        if not instance.private:
```

Comment:
This makes sense, I'd probably add a small comment on what this does (only allows setting private to true).